friture/audioproc.py

```diff
@@ -95,6 +100,17 @@ def update_freq_cache(self):
             self.B = 0.17 + 20. * log10(Rb + eps)
             self.A = 2.0 + 20. * log10(Ra + eps)
 
+            self.recalculate_calibration()
+    
+    def recalculate_calibration(self):
+        try:
+            if hasattr(self.parent_widget.parent(), "dockmanager"):
```

Comment:
This creates a strong coupling with how the audioproc is used in the object chain. It would be great to find a way to pass the calibration without having to look at the parents